### ISSUE TITLE
How is this different from Neovim's native lsp hover?

### ISSUE THREAD

**Issue by Ciel-MC** [LukasPietzschmann/boo.nvim#2]

**Comment by LukasPietzschmann** [comment]
Funnily enough, I got asked this on [reddit](https://www.reddit.com/r/neovim/comments/1b9yhiq/comment/ktz85tz), too :D

To be honest, when writing this plugin, I did not know about NeoVim's native LSP hover (which makes me feel a bit dumb right now).

But I plan to extend the information that this plugin shows beyond what the language server provides. This could be helpful if, e.g., there is no LSP attached, or you want to have a single UI to show infos from various sources.

TL;DR: Currently there's no difference (besides maybe some settings), but there will be in the future.

**Comment by LukasPietzschmann** [comment]
@Ciel-MC I'm closing this for now. If you've got any more questions, feel free to just reopen this issue :)
